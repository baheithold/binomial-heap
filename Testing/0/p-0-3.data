2 1 371 ; 5 5 300 ; 5 4 417 ; 3 7 67 ; 7 1 68 ; 
0 0 291 ; 3 5 480 ; 3 5 9 ; 3 1 488 ; 1 6 324 ; 

