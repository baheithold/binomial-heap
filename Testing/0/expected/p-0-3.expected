0: 0
1: 1(0)58
weight: 58
